### ISSUE TITLE
Public uPheno2 (Alpha) review

### ISSUE THREAD

**Issue by matentzn** [obophenotype/upheno-dev#26]
We have published a first alpha of uPheno2 here:https://doi.org/10.5281/zenodo.3710690

I need you all now to open it in Protege and take a look, especially at your own branches in there, and tell me whether it looks reasonable or not. Any observations, wishes and criticisms are welcome.

**Comment by drseb** [comment]
This class should probably renamed to abnormal shape of anatomical entity

<img width="322" alt="Screenshot 2020-03-14 at 15 17 46" src="https://user-images.githubusercontent.com/5172897/76683790-0c933680-6607-11ea-947d-7c3a44a89ad9.png">

**Comment by sbello** [comment]
A couple of behavior branch questions, based off this image
![upheno_volun_movement](https://user-images.githubusercontent.com/10979793/76789565-79d3d280-6793-11ea-899e-56329b158514.png)
1. Why is only the HPO branch under the Upheno terms?
2. Why are there 2 terms with the same name but different IDs (gray arrows)?
3. The organization seems off to me, the parent term is "abnormal voluntary movement behavior" but one of the child terms is "involuntary movements" (rad arrow) and the HPO term 'Abnormality of movement" would also seem broader than voluntary movement.

**Comment by sbello** [comment]
Unclear why the MP and XPO 'abnormal spermatid morphology' not grouped under the Upheno 'abnormal morphology of spermatid' term? Both have EQs.
![spermatid2](https://user-images.githubusercontent.com/10979793/76793244-1c438400-679b-11ea-8d7f-31abd1af0389.png)

**Comment by ddooley** [comment]
Can labelling be made a bit clearer in the maturity and life stage classes?  Can they be merged, with a few more parent classes to encompass the zebrafish life stage differentiae?

![image](https://user-images.githubusercontent.com/4000582/77194352-084e9980-6a9d-11ea-883f-5b2e040a39a1.png)

and

![image](https://user-images.githubusercontent.com/4000582/77194621-89a62c00-6a9d-11ea-8b79-1ba988ec457a.png)

Cheers,

Damion

**Comment by matentzn** [comment]
@ddooley Since upheno is a derived product (essentially a merge of many others), it wont be possible to change the labels of our source ontologies! To fix those, you would have to ask them directly. Once they are fixed in the source (you can guess the source from the IRI of the life stage you are looking at), they will be fixed in uPheno.

Life stages are a bit behind on integration I have to admit..  Maybe you should talk to https://github.com/obophenotype/developmental-stage-ontologies ?

**Comment by ddooley** [comment]
Ok. I ended up finding https://github.com/obophenotype/uberon/wiki/Modeling-developmental-stages which seems to put a framework around what different ontologies should be doing re. stages. It does raise a question: Should PATO be talking about life stage as a quality rather than a processual entity?  But I'll set that aside for now. For my own purposes, what UBERON has in "life cycle stage" is sufficient.  Thx.

**Comment by mellybelly** [comment]
@ddooley yes please make lifestage requests in the Uberon tracker, we keep the "uber" stage ontology there.
